bundles/test/org.eclipse.smarthome.test/src/main/java/org/eclipse/smarthome/test/java/JavaOSGiTest.java

```diff
@@ -64,7 +64,7 @@ public void bindBundleContext() {
      *
      * @return bundle context
      */
-    private BundleContext getBundleContext() {
+    public BundleContext getBundleContext() {
```

Comment:
I would prefer if you change the bundleContext member from default to protected.
getBundleContext() is used to initialize the bundleContext variable only once (I know the description is not correct).